addon/ng2/blueprints/ng2/files/src/client/app/__name__.component.ts

```diff
@@ -1,17 +1,13 @@
 import {Component} from 'angular2/core';
-import {RouteConfig, ROUTER_DIRECTIVES, ROUTER_PROVIDERS} from 'angular2/router';
```

Comment:
This component still has a `router-outlet`, which means it needs `ROUTER_DIRECTIVES` I believe.